mod.ts

```diff
@@ -11,4 +11,4 @@ export {
   type QueueJob,
   type QueueJobId,
 } from "./src/types.ts";
-export { useQueue } from "./src/use-queue.ts";
+export { JetQueueMode, setQueueMode, useQueue } from "./src/use-queue.ts";
```

Comment:
这里不需要导出这两个东西，只有测试会用上
```suggestion
export { useQueue } from "./src/use-queue.ts";
```